.gitignore

```diff
@@ -49,6 +49,7 @@ src/qt/test/moc*.cpp
 .libs
 .*.swp
 *.*~*
+*.backup
```

Comment:
Not only that, but we shouldn't be creating a `wallet.backup` file in the src directory at all. It should go in the temporary directory created for the other test files.

Sounds like a really bad thing now, someone could confuse it for a real wallet backup, for one.

Thanks for noticing @eklitzke.